packages/react-integration/demo-app-ts/tsconfig.json

```diff
@@ -18,6 +18,14 @@
     "noEmit": true,
     "jsx": "preserve",
     "strict": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "noImplicitAny": true,
+    "alwaysStrict": true,
+    "strictBindCallApply": true,
+    "strictPropertyInitialization": false,
+    "strictNullChecks": false,
+    "strictFunctionTypes": false,
```

Comment:
> Enabling `--strict` enables --noImplicitAny, --noImplicitThis, --alwaysStrict, --strictBindCallApply, --strictNullChecks, --strictFunctionTypes and --strictPropertyInitialization.

If you could remove the following redundant flags that'd be great:
- noImplicitAny
- noImplicitThis
- alwaysStrict
- strictBindCallApply

Reply:
done